Cargo.toml

```diff
@@ -17,3 +17,7 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 spinners = "4.1.0"
 serde_yaml = "0.8"
+dirs = "5.0.1"
+config = "0.14.0"
+toml = "0.8.12"
+tempfile = "3.10.1"
```

Comment:
❓ So my understanding is that this is only used for the tests?

Reply:
tempfile is just used for the tests yes. that okay as a dependency?

Reply:
Yeah, should be fine, just double checking that my understanding was correct.